### DISCUSSION TITLE
Implementing Scout pipeline in clinical practice

### DISCUSSION THREAD

**Discussion by JenkoB** [Clinical-Genomics/scout#3743] [Q&A]
Hello,

we are very interested to implement Scout in our clinical pipeline. It has very nice functionality for clinical practice. 
However we couldn't find any module for data input. What is the way to input cases info (so to create yaml file) and users?

We would be very grateful for help and advice how to inplement scout in our lab.

Kind regards,

Barbara

**Comment by dnil** [comment]
Dear Barbara, 

that sounds lovely! We have found it very useful over the years, and are happy to help as far as we can. The current local Stockholm instance hosts well over 10k cases, and it is in use on a handful of other sites around Sweden and the globe.

Sweden is a bit into the holiday groove already, so I'm afraid we won't be able to offer much assistance over the next couple of weeks. Decoration of the 🎄 is about to commence.  Normality resumes sometime early to mid January. 😅  

In the meantime, you can find some tips in the admin and user documentation sections: https://clinical-genomics.github.io/scout/admin-guide/loading-users/ and https://clinical-genomics.github.io/scout/admin-guide/loading-case/. The trick is that basically all admin stuff is done via the command line interface. This can be anywhere really, as long as it has some way to access to the mondgodb, either from a docker/singularity container or installed in a virtual environment on your personal machines or on a server. 

For testing out, I would suggest just manually creating a few very basic load yaml configs. They don't need to contain very much more than sample names, VCF and perhaps bam/cram file names: the rest is very much optional.

The next step would be to write a small script extracting the same things from wherever you store them, like a LIMS, db, or such.

Locally we use an automated service for ordering and uploading, talking to the LIMS etc, but I'm afraid it will likely be too strictly tied to our operation to be super useful: https://github.com/Clinical-Genomics/cg.

Maybe we could set up a short video conference in January and we can do a q&a with whatever you are still wondering? Cheers, and happy holidays!

**Comment by JenkoB** [comment]
Dear Daniel,

thank you a lot for the fast and explainable answer.

As you suggest, I will firstly create a basic .yaml config case from our data and later write a script or service to do and load it automatically. I will also check your system (https://github.com/Clinical-Genomics/cg.) to get a better idea.

It would be a great help to us to set up a short video conference in January for additional discussion. We can discuss the best day and time for it after the holidays.

Cheers, and happy holidays!

Barbara
